### ISSUE TITLE
Map collection: World, Robinson projection - doesn't support lat/lon

### ISSUE THREAD

**Issue by KacperMadej** [highcharts/highcharts#4869]
All 3 maps of World, Robinson projection doesn't seem to support lat/lon.

Example with `world-robinson-lowres`: http://jsfiddle.net/wsysowyt/
(Example made from [this demo](http://www.highcharts.com/maps/demo/latlon-advanced) )

**Comment by oysteinmoseng** [comment]
That is correct, and caused by Proj4js not supporting the Robinson projection. We should periodically check for updates on that.

**Comment by tomascassidy** [comment]
Can't see robinson anywhere in https://github.com/proj4js/proj4js/tree/master/lib/projections so I'm guessing it's not supported in Proj4js yet.

**Comment by tomascassidy** [comment]
https://github.com/proj4js/proj4js/issues/161

**Comment by calvinmetcalf** [comment]
pull requests accepted guys, especially if accompanied by lots of test

**Comment by KacperMadej** [comment]
Version 2.4.4 (current) still doesn't resolve the problem, but there's a promising commit that might be added to next version - https://github.com/proj4js/proj4js/commit/c72ea139684592d19f79436b6604ab0282783d3b
